widgetssdk/src/main/java/com/glia/widgets/core/engagement/GliaEngagementTypeRepository.java

```diff
@@ -2,6 +2,7 @@
 
 import com.glia.widgets.core.operator.GliaOperatorMediaRepository;
 import com.glia.widgets.core.visitor.GliaVisitorMediaRepository;
+import com.glia.widgets.helper.Logger;
```

Comment:
As far as I understand, this import is not used, is it?

Reply:
Yes, thank you!